prune_test.go

```diff
@@ -193,26 +193,51 @@ func (s *PruneSuite) TestMultipleCollections(c *gc.C) {
 	s.assertTxns(c, lastTxnIds...)
 }
 
-func (s *PruneSuite) TestWithStash(c *gc.C) {
+func (s *PruneSuite) TestWithStashReference(c *gc.C) {
 	// Ensure that txns referenced in the stash are not pruned from
 	// the txns collection.
 
-	// An easy way to get something into the stash is to delete a
-	// document.
-	txnId0 := s.runTxn(c, txn.Op{
+	// An easy way to get something into the stash is to create a document, but have the transaction fail
```

Comment:
I couldn't find a way of creating a deleted document that would have a reference to a transaction that wasn't completed.
I looked around for chaos() points that would be "the document has been put into the stash for removal but has not been committed", but couldn't find a chaos point for it.